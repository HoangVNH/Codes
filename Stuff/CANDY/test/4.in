10
3
VDX
7
VVVDDDD
3
XVV
7
VVVVVVV
4
VVDD
4
XXXX
9
DDXVVVVVV
10
DDXXXXXXVV
6
XVXXXX
8
XVVXXXXX
