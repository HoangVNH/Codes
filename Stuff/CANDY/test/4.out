1
6
2
0
2
0
8
8
5
6
